CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## master (unreleased)
 
+* [#181](https://github.com/rubocop-hq/rubocop-performance/pull/181): Change default configuration for `Performance/CollectionLiteralInLoop` to `Enabled: 'pending'`. ([@ghiculescu][])
```

Comment:
This entry looks good! Only one more point, can you move the entry to under the `### Changes` section?

Reply:
ooops, done.

Reply:
Sorry about that.

Reply:
Thank you for updating!